travis.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+build_cpp() {
+  ./autogen.sh
+  ./configure
+  make -j2
+  make distcheck -j2
```

Comment:
`make distcheck` doesn't need the normal build, but the conformance test does. (In theory I suppose the conformance test could be split out separately, but that seems like a bit overkill for now, I think?)